style.css

```diff
@@ -0,0 +1,383 @@
+.container{
+      
+      display:grid;
+      grid-template-columns: .9fr 1.25fr 0.8fr .9fr   ;
+      grid-template-rows:  1fr 1fr 1fr 0.7fr 2.5fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+      grid-gap: 45px;
+      grid-template-areas: 
+
+      
+      "sidebar-blank1 header header sidebar-blank2  "
+      "sidebar-blank1 sub-header sub-header sidebar-blank2 "
+      "sidebar-blank1 sub-header sub-header sidebar-blank2 "
+      "sidebar-blank1 sub-header sub-header sidebar-blank2  "
+      "content1 content1 content1 content1  "
+      "sidebar-blank3 blog note sidebar-blank4"
+      "sidebar-blank3 blog note sidebar-blank4"
+      "sidebar-blank3 blog note sidebar-blank4"
+      "sidebar-blank3 blog note sidebar-blank4"
+      "sidebar-blank3 blog note sidebar-blank4"
+      "sidebar-blank3 blog note sidebar-blank4"
+      "sidebar-blank3 blog note sidebar-blank4"
+      "sidebar-blank3 blog note sidebar-blank4"
+       ;
+}
```

Comment:
instead of using 'sidebar-blank1' and such, you could just replace them with a '.' to show that "this grid section is filled by nothing